baseband/mark5b/tests/test_mark5b.py

```diff
@@ -528,6 +528,16 @@ def test_filestreamer(self, tmpdir):
             assert np.all(fhs.read(20000) == record[:, 0])
             assert np.all(fhns.read(20000) == record[:, 0])
 
+        # Test that sample_rate can be inferred from max frame number
+        m5_test_samplerate = str(tmpdir.join('test_samplerate.m5b'))
+        with mark5b.open(m5_test_samplerate, 'ws', time=start_time, nchan=8, 
```

Comment:
It is actually a bit weird that this works, as the `frame_nr` is inconsistent with the `start_time` (which is exactly at 0 seconds, so the frame number should be 0). That is a separate bug: #158. But it would be good to ensure that this test is actually correct. At the same time, we might as well use a sample rate that is actually possible, say 1 MHz, which implies a frame rate of 200 Hz...  I'll make a quite correction mysef... (and also remove the extraneous space that causes the test failure).